### PR TITLE
Blog and pagination jisc styled

### PR DESCRIPTION
**Pull request by onepablo**
There's a couple of "could haves" outstanding but let's just role this out.

Known issues:
- the breadcrumbs need the correct html rather than that generated by the plugin
- we're using the wrong "fat footer" (R&D)
- blog index could do with a little (little) more work (why is the <h1> underlined basically)

### CODE REVIEW

File: wp-content/themes/twentysixteen-child/css/admin.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.col.pull-1{
```
Comment (adam-rehin) [inline wp-content/themes/twentysixteen-child/css/admin.css]: What is admin.css for? Where is it used?

File: wp-content/themes/twentysixteen-child/single.php [hunk 2]
```diff
@@ -7,41 +7,21 @@
  * @package Jisc
```
Comment (adam-rehin) [inline wp-content/themes/twentysixteen-child/single.php]: Doesn't the Single post page need to show next/prev links?  Or is that not part of Jisc way of displaying blog posts? Or am I misunderstanding single.php?

Comment (onepablo) [inline wp-content/themes/twentysixteen-child/single.php]: I actually don’t know and wondered that myself when I removed it.  Some of the “web team” have just emerged so I shall find out for sure…

Comment (onepablo) [inline wp-content/themes/twentysixteen-child/single.php]: Right, there’s no requirement to have this functionality.  However in discussions they’ve decided that, if implemented it shouldn’t simply be “previous” and “next” (with titles) but rather the prev/next posts in that category.

I say we leave it out unless requested.

**Comment by adam-rehin** [comment]
.site-footer {
		/* JISC
		-webkit-align-items: center;
		-ms-flex-align: center;
		align-items: center;
		display: -webkit-flex;
		display: -ms-flexbox;
		display: flex;
		-webkit-flex-wrap: wrap;
		-ms-flex-wrap: wrap;
		flex-wrap: wrap;
		padding: 0 4.5455% 3.5em;
		*/
		padding:0;
	}

**Comment by adam-rehin** [comment]
Sorry - I didn't mean to close it!

I meant to say that I've sent a comment via Skype chat about style.css (as no way of doing it on the comparison pg).

**Pull request merged**